clang/lib/CIR/Dialect/Transforms/TargetLowering/Targets/X86_64ABIInfo.h

```diff
@@ -69,6 +69,14 @@ class X86_64ABIInfo : public cir::ABIInfo {
                                     mlir::Type SourceTy,
                                     unsigned SourceOffset) const;
 
+  /// getIndirectResult - Give a source type \arg Ty, return a suitable result
```

Comment:
We should leave comments repeating names out like in `getIndirectResult - `, LLVM is moving away from it for some time, we don't want to revisit this during upstreaming

Reply:
If I read correctly, you're saying to remove the names? If yes, done.

Reply:
Yep, thanks